docs/templates/e-commerce.rst

```diff
@@ -0,0 +1,24 @@
+.. _e-commerce:
+
+===========================================
+E-commerce spider template (``e-commerce``)
```

Comment:
```suggestion
E-commerce spider template (``ecommerce``)
```

Likely affects other parts of the change.

Reply:
Is it just a suggested change because you aren't sure?

Reply:
Because I realized this later, when working on docs.zyte.com, and added it here as a note to myself to fix that as I apply the rest of the feedback here later. I did not want to let this distract myself from docs.zyte.com work, and I wanted to check also if I left other references to the spider ID elsewhere.